api/autopilot.go

```diff
@@ -88,13 +88,21 @@ type (
 		Configured         bool        `json:"configured"`
 		Migrating          bool        `json:"migrating"`
 		MigratingLastStart TimeRFC3339 `json:"migratingLastStart"`
+		Pruning            bool        `json:"pruning"`
+		PruningLastStart   TimeRFC3339 `json:"pruningLastStart"`
 		Scanning           bool        `json:"scanning"`
 		ScanningLastStart  TimeRFC3339 `json:"scanningLastStart"`
 		UptimeMS           DurationMS  `json:"uptimeMS"`
 
 		StartTime time.Time `json:"startTime"`
 		BuildState
 	}
+
+	// PruningStatsResponse is the response type for the
+	// /autopilot/stats/pruning endpoint.
+	PruningStatsResponse struct {
+		AvgPruningSpeedMBPS map[string]float64 `json:"avgPruningSpeedMBPS"`
```

Comment:
I considered using the new metrics database but I'm not really sure if it fits 100%. Figured adding a `/stats` endpoint is also ok. If not we should probably get rid of all `/stats` endpoint, ideally before 1.0 still to avoid the compat issue.